styles.css

```diff
@@ -0,0 +1,540 @@
+    display: inline-block; 
+    width: 40%; 
+}
+
+.container{
+    height: 100vh; 
+    color: white; 
+    display: inline-block;
+    align-items: center; 
+    justify-content: center; 
+    margin-right: 500px;
+    margin-top: 660px;
+    position: absolute; 
+    width: 40%; 
+}
+
+.container h2{
+    font-size: 38px; 
+    font-weight: bold; 
+}
+
+span{
+    color:#FFB6C1; 
+}
+
+.scroll-text{
+    color: white; 
+    display: inline-block; 
+    rotate: -90deg;
+    margin-bottom: 50px;
+    font-weight: 500;
+}
+
+.scroll{
+    position: absolute;  
+    transform: translate(-50%,-50%);
+    align-items: center; 
+    justify-content: center; 
+    margin-top: 620px; 
+    margin-right: 1000px; 
+
+}
+
+.scroll span{
+    display: block;
+    width: 10px; 
+    height: 10px; 
+    border-bottom: 1px solid white; 
+    border-right: 1px solid white; 
+    transform: rotate(45deg); 
+    margin: -10px; 
+    animation: animate 2s infinite; 
+    margin-left: 49px; 
+}
+
+.scroll span:nth-child(2){
+    animation-delay: -0.2s; 
+    transform: rotate(45deg) translate(-20px,-20px); 
+}
+
+.scroll span:nth-child(3){
+    animation-delay: -0.4s; 
+}
+
+@keyframes animate{
+    0%{
+        opacity: 0; 
+        transform: rotate(45deg) translate(-20px,-20px); 
+    }
+    50%{
+        opacity: 1; 
+    }
+    100%{
+        opacity: 0; 
+        transform: rotate(45deg) translate(20px,20px); 
+    }
+}
+
+.wave{
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    overflow: hidden;
+    line-height: 0;
+    transform: rotate(180deg);
+    z-index: 40;
+    margin-bottom: -0.5px;  
+}
+
+.wave svg{
+    position: relative;
+    display: block;
+    width: calc(120% + 1.3px);
+    height: 85px;
+    transform: rotateY(180deg);
+}
+
+.wave .shape-fill{
+    fill: #FFE4E1;
+}
+
+.grid ul{
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+  
+.About{
+    background-color: #FFE4E1;
+    text-align: center;
+    padding: 4rem 25rem; 
+}
+
+h3 span{
+    color: black;
+    font-size: 24px; 
+    text-align: center;
+    font-weight: 700;
+    line-height: 1.34rem;
+}
+
+.grid{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+    justify-content: center;
+    width: min(640px, 85vw);
+    margin-inline: auto;
+}
+
+.circle-1{
+    height: 100px; 
+    width: 100px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-top: -300px; 
+    margin-left: 200px; 
+}
+
+.circle-2{
+    height: 200px; 
+    width: 200px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-left: 950px; 
+    margin-top: -400px; 
+}
+
+.circle-3{
+    height: 300px; 
+    width: 300px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-top: -100px; 
+    margin-left: -500px; 
+}
+
+.circle-4{
```

Comment:
The circle on the top left (circle-4) is getting cut off too early. How can I get this circle on top of the pink wave without having it overlap into the welcome-section?